### PR TITLE
cambios al hola mundo

### PR DESCRIPTION
**Pull request by jhonlopezr**

### CODE REVIEW

**Review by daesloard — APPROVED** [review]

**Review by daesloard — APPROVED** [review]

**Comment by jhonlopezr** [comment]
Review OK

**Review by jhonlopezr — COMMENTED** [review]
Changes "Hola, mundo"

File: hola.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log("hola")
+console.log("hola, putos")
```
Comment (jhonlopezr) [inline hola.js:1]: CHanges completed

**Review by jhonlopezr — COMMENTED** [review]
Finish

**Pull request closed without merging**